*** Begin Patch
diff --git a/crates/gpui/src/gpui.rs b/crates/gpui/src/gpui.rs
index 8398874ea257a4..0b0e2c3257b6c8 100644
--- a/crates/gpui/src/gpui.rs
+++ b/crates/gpui/src/gpui.rs
@@ -10,10 +10,8 @@ extern crate self as gpui;
 mod action;
 mod app;
 
-mod arena;
 mod asset_cache;
 mod assets;
-mod bounds_tree;
 mod color;
 /// The default colors used by GPUI.
 pub mod colors;
@@ -22,10 +20,13 @@ mod elements;
 mod executor;
 mod platform_scheduler;
 pub(crate) use platform_scheduler::PlatformScheduler;
+mod fiber;
 mod geometry;
 mod global;
+mod identity;
 mod input;
 mod inspector;
+mod intrinsic_size;
 mod interactive;
 mod key_dispatch;
 mod keymap;
@@ -35,6 +36,7 @@ pub mod prelude;
 mod profiler;
 #[cfg(any(target_os = "windows", target_os = "linux"))]
 mod queue;
+mod render_node;
 mod scene;
 mod shared_string;
 mod shared_uri;
@@ -44,6 +46,7 @@ mod subscription;
 mod svg_renderer;
 mod tab_stop;
 mod taffy;
+mod transform;
 #[cfg(any(test, feature = "test-support"))]
 pub mod test;
 mod text_system;
@@ -73,7 +76,6 @@ mod seal {
 pub use action::*;
 pub use anyhow::Result;
 pub use app::*;
-pub(crate) use arena::*;
 pub use asset_cache::*;
 pub use assets::*;
 pub use color::*;
@@ -81,12 +83,15 @@ pub use ctor::ctor;
 pub use element::*;
 pub use elements::*;
 pub use executor::*;
+pub(crate) use fiber::*;
 pub use geometry::*;
 pub use global::*;
 pub use gpui_macros::{AppContext, IntoElement, Render, VisualContext, register_action, test};
 pub use http_client;
+pub(crate) use identity::*;
 pub use input::*;
 pub use inspector::*;
+pub(crate) use intrinsic_size::*;
 pub use interactive::*;
 use key_dispatch::*;
 pub use keymap::*;
@@ -96,6 +101,7 @@ pub use profiler::*;
 #[cfg(any(target_os = "windows", target_os = "linux"))]
 pub(crate) use queue::{PriorityQueueReceiver, PriorityQueueSender};
 pub use refineable::*;
+pub(crate) use render_node::*;
 pub use scene::*;
 pub use shared_string::*;
 pub use shared_uri::*;
@@ -105,8 +111,8 @@ pub use styled::*;
 pub use subscription::*;
 pub use svg_renderer::*;
 pub(crate) use tab_stop::*;
-use taffy::TaffyLayoutEngine;
 pub use taffy::{AvailableSpace, LayoutId};
+pub use transform::*;
 #[cfg(any(test, feature = "test-support"))]
 pub use test::*;
 pub use text_system::*;
*** End Patch
